2 - RTB/Documentazione esterna/Verbali/23-12-15/23-12-15.typ

```diff
@@ -0,0 +1,59 @@
+#import "/template.typ": *
+
+#show: project.with(
+  date: "15/12/23",
+  subTitle: "",
+  docType: "verbale",
+  authors: (
+    "Nardo Silvio",
+  ),
+  externalParticipants : (
+    (name: "Alberton Michela", role: "Membro azienda proponente"),
+    (name: "Bassani Matteo", role: "Referente aziendale"),
+    (name: "Ruffin Filippo", role: "Membro azienda proponente"),
+  ), 
+  timeStart: "12:00",
+  timeEnd: "12:30",
+);
+
+= Ordine del giorno
+Ai referenti aziendali sono stati esposti i progressi sui PoC rispetto al precedente meeting esterno e sono sorte discussioni relativamente a:
```

Comment:
Il referente aziendale è uno solo. Rimuoverei in toto l'incipit della frase, tanto è sottointeso chi siano gli interlocutori